AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -141,6 +142,43 @@ class CompleteScreenViewModel: ObservableObject {
         exitRoute()
     }
     
+    private func getAndDisplayData() {
+        if session.stream.first!.sensorName.contains("AirBeam") {
+            do {
+                try streamsDownloader.downloadStreams(with: session.id, for: .airBeam3) { result in
+                    switch result {
+                    case .success(let success):
+                        // TODO: - FIX Thresholds and Improve this part of code
+                        // TODO: Compare strings and get threshold value, example - comment below
+                        // let vl = MeasurementStream.init(sensorName: .pm1, sensorPackageName: "").thresholdVeryLow
```

Comment:
I don't fully understand this comment but we shouldn't read threshold values from database at this stage. We should use the threshold values that we get from backend